src/main/java/com/car/app/controllers/CarDao.java

```diff
@@ -1,6 +1,5 @@
-package com.car.app.service;
+package com.car.app.controllers;
```

Comment:
Usually the package name is in singular, unless is something really related to the domain of the application.
Could you please, rename the package to singular ?